app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,10 +29,22 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * Returns all products in this application.
+     *
+     * @return all products.
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * Returns the product with given ID.
+     *
+     * @param id is the identifier of the product.
+     * @return the product with given ID.
+     * @throws ProductNotFoundException in case any product doesn't exist with the given ID.
```

Comment:
`@` 태그에서는 마침표를 생략합니다. 이렇게 하면 어떨까요?

```suggestion
     * Returns a product with the given ID.
     *
     * @param id is the identifier of the product
     * @return a product
     * @throws ProductNotFoundException if a product cannot be found
```